src/Console/Arguments.php

```diff
@@ -142,7 +142,7 @@ public function getOptions(): array
      * @param string $name The name of the option to check.
      * @return string|int|bool|null The option value or null.
      */
-    public function getOption(string $name)
+    public function getOption(string $name): string|int|bool|null
```

Comment:
If default values can only be strings should this also include `int` or should we remove `int` and have userland code convert/cast?

Reply:
Seems like we should remove it to align with default. The parsing code also re-casts the default to a string.

https://github.com/cakephp/cakephp/blob/5.0/src/Console/ConsoleOptionParser.php#L895

I'm not sure there is actually an "int" anywhere in the parsed option values. We get strings from php, and I don't see it converted anywhere.